lib/interface.rb

```diff
@@ -0,0 +1,81 @@
+require_relative 'menu'
+require_relative 'order'
+
+class Interface
+  def initialize(menu = Menu.new, order = Order.new)
+    @menu = menu
+    @order = order
+    puts "Welcome to Ewan's pizzeria!"
```

Comment:
no - do not put presentation strings in your business logic, especially in the initialize method.